src/views/CVERecord/SearchResults.vue

```diff
@@ -253,10 +226,15 @@
 <script setup>
 import { useCveListSearchStore } from '@/stores/cveListSearch';
 import { usePartnerStore } from '@/stores/partners';
-import { watch } from 'vue';
+import { createApp, watch } from 'vue';
 import { useRouter } from 'vue-router';
+import ServiceUnavailable from '@/components/ServiceUnavailable.vue'
+
 const cveListSearchStore = useCveListSearchStore();
 const router = useRouter();
+const app = createApp({});
+
+app.component('ServiceUnavailable', ServiceUnavailable);
```

Comment:
I don't have a great understanding of Vue3 yet, but should this be globally registered in main.ts instead? I think a second `app` is being created, since one is already created in main.ts.